libs/velo-external-db-core/src/router.ts

```diff
@@ -276,8 +276,8 @@ export const createRouter = () => {
 
         const { collectionIds } = req.body
         try {
-            const data = await schemaService.list(collectionIds)
-            streamCollection(data.collection, res)
+            const collectionsList = await schemaService.list(collectionIds)
+            streamCollection(collectionsList, res)
```

Comment:
should stream array of 
`[{collection: Collection}, {collection: Collection}, ....]` look on query for example